lib/portal

```diff
@@ -13,6 +13,23 @@ function install_mariadb {
 
 # install_portal() - Function that pulls and install the source code of Portal
 function install_portal {
+
+if [[ "$build_image" == "True" ]]; then
+    # Build locally
+    clone_repo portal /opt/portal
```

Comment:
What about to move this line, in that way the order of execution should be as follows

* install docker
* remove old images
* clone repos (portal, ecompsdkos and dmaapbc)